### ISSUE TITLE
Expose the radiometric and shape weights to the user [Feat] 

### ISSUE THREAD

**Issue by pierreroudier** [OSGeo/grass#3392]
In the current documentation of `i.segment`, it reads that:

> Both Euclidean and Manhattan distances use the normal definition, considering each raster in the image group as a dimension. In future, the distance calculation will also take into account the shape characteristics of the segments. The normal distances are then multiplied by the input radiometric weight. Next an additional contribution is added: (1-radioweight) * {smoothness * smoothness weight + compactness * (1-smoothness weight)}, where compactness = Perimeter Length / sqrt( Area ) and smoothness = Perimeter Length / Bounding Box. 

My understanding is that in its current state, `radio_weight = 1`, and therefore `smoothness_weight` is ignored.

I do see in the code, though, that those parameters seem [to be possible to set](https://github.com/OSGeo/grass/blob/c099a3278c90cebebf31a362361140193ed1721c/imagery/i.segment/parse_args.c#L281C1-L296C7), and that [a function to calculate the shape index exists](https://github.com/OSGeo/grass/blob/c099a3278c90cebebf31a362361140193ed1721c/imagery/i.segment/region_growing.c#L859-L935). Sorry, @metzm, unfortunately I do not know C enough to investigate much further than this! 

Is there a way to expose and use the two parameters `radio_weight` and `smooth_weight`? At the moment it seems to be a significant limitation of `i.segment`, in particular in situations where strong linear features exists, it makes it hard to prevent `i.segment` to create "spaghetti" long, thin, elongated regions.

**Comment by metzm** [comment]
You are correct, in its current state, `radio_weight = 1`, and therefore `smoothness_weight` is ignored because using these region shape indices would substantially slow down the module. The relevant code exists but is deactivated. The module is as it is now relatively slow and needs a lot of computing resources (space and time), therefore I decided to deactivate these features regarding region shapes which would further substantially slow down the module.
